R/Lithuania.R

```diff
@@ -0,0 +1,469 @@
+        "\u0160iauli\u0173 r. sav.", "LT-44", "\u0160iauli\u0173 apskritis", "LT-SA", "\u0160iauliai County", "district municipality",
+        "\u0160iauli\u0173 m. sav.", "LT-43", "\u0160iauli\u0173 apskritis", "LT-SA", "\u0160iauliai County", "city municipality",
+        "Jurbarko r. sav.", "LT-12", "Taurag\u0117s apskritis", "LT-TA", "Taurag\u0117 County", "district municipality",
+        "Pag\u0117gi\u0173 sav.", "LT-29", "Taurag\u0117s apskritis", "LT-TA", "Taurag\u0117 County", "municipality",
+        "\u0160ilal\u0117s r. sav.", "LT-45", "Taurag\u0117s apskritis", "LT-TA", "Taurag\u0117 County", "district municipality",
+        "Taurag\u0117s r. sav.", "LT-50", "Taurag\u0117s apskritis", "LT-TA", "Taurag\u0117 County", "district municipality",
+        "Ma\u017eeiki\u0173 r. sav.", "LT-26", "Tel\u0161i\u0173 apskritis", "LT-TE", "Tel\u0161iai County", "district municipality",
+        "Plung\u0117s r. sav.", "LT-35", "Tel\u0161i\u0173 apskritis", "LT-TE", "Tel\u0161iai County", "district municipality",
+        "Rietavo sav.", "LT-39", "Tel\u0161i\u0173 apskritis", "LT-TE", "Tel\u0161iai County", "municipality",
+        "Tel\u0161i\u0173 r. sav.", "LT-51", "Tel\u0161i\u0173 apskritis", "LT-TE", "Tel\u0161iai County", "district municipality",
+        "Anyk\u0161\u010di\u0173 r. sav.", "LT-04", "Utenos apskritis", "LT-UT", "Utena County", "district municipality",
+        "Ignalinos r. sav.", "LT-09", "Utenos apskritis", "LT-UT", "Utena County", "district municipality",
+        "Mol\u0117t\u0173 r. sav.", "LT-27", "Utenos apskritis", "LT-UT", "Utena County", "district municipality",
+        "Utenos r. sav.", "LT-54", "Utenos apskritis", "LT-UT", "Utena County", "district municipality",
+        "Visagino sav.", "LT-59", "Utenos apskritis", "LT-UT", "Utena County", "municipality",
+        "Zaras\u0173 r. sav.", "LT-60", "Utenos apskritis", "LT-UT", "Utena County", "district municipality",
+        "Elektr\u0117n\u0173 sav.", "LT-08", "Vilniaus apskritis", "LT-VL", "Vilnius County", "municipality",
+        "\u0160al\u010dinink\u0173 r. sav.", "LT-42", "Vilniaus apskritis", "LT-VL", "Vilnius County", "district municipality",
+        "\u0160irvint\u0173 r. sav.", "LT-47", "Vilniaus apskritis", "LT-VL", "Vilnius County", "district municipality",
+        "\u0160ven\u010dioni\u0173 r. sav.", "LT-49", "Vilniaus apskritis", "LT-VL", "Vilnius County", "district municipality",
+        "Trak\u0173 r. sav.", "LT-52", "Vilniaus apskritis", "LT-VL", "Vilnius County", "district municipality",
+        "Ukmerg\u0117s r. sav.", "LT-53", "Vilniaus apskritis", "LT-VL", "Vilnius County", "district municipality",
+        "Vilniaus m. sav.", "LT-57", "Vilniaus apskritis", "LT-VL", "Vilnius County", "city municipality",
+        "Vilniaus r. sav.", "LT-58", "Vilniaus apskritis", "LT-VL", "Vilnius County", "district municipality",
+        "Unknown", NA, "Unknown", NA, NA, NA
+      )
```

Comment:
If this is the same as whats in `region_codes` could that not be called instead? rather than having it two places?

Reply:
I am going to work on using that (which tidies the code too).